glvis.cpp

```diff
@@ -56,11 +57,15 @@ string      extra_caption;
 
 // Global variables
 int input = 1;
-Mesh *mesh = NULL;
-Vector sol, solu, solv, solw, normals;
-GridFunction *grid_f = NULL;
-int is_gf = 0;
-string keys;
+Mesh *&mesh = stream_state.mesh;
```

Comment:
Are there any methods left in glvis.cpp that rely on these variables, or could we just pass around the `stream_state` object?

Reply:
Yeah, at least `GLVisInitVis`, `GLVisStartVis`, `ExecuteScriptCommand`, and `PlayScript`.  We could switch to using an object but that's a bigger diff and I think it would be better for a followup PR.